internal/bgp/frrk8s/frrk8s.go

```diff
@@ -233,6 +235,10 @@ func (sm *sessionManager) updateConfig() error {
 			}
 			portUint16 := uint16(portUint)
 
+			if reflect.DeepEqual(s.PasswordRef, corev1.SecretReference{}) && s.Password != "" {
```

Comment:
My idea was to support both and have the password translated to a secret, but it's also true that a smoother approach would be just to add plain passwords support in frrk8s.
I am not sure I like this, because it would require users to change their configurations if they want to switch to the new mode.